bokeh/util/callback_manager.py

```diff
@@ -53,7 +59,13 @@
 # General API
 #-----------------------------------------------------------------------------
 
-EventCallback = Callable[[Event], None]
+# TODO (bev) the situation with no-argument Button callbacks is a mess. We
+# should migrate to all callbacks receving the event as the param, even if that
+# means auto-magically wrapping user-supplied callbacks for awhile.
+EventCallbackWithEvent = Callable[[Event], None]
+EventCallbackWithoutEvent = Callable[[], None]
+EventCallback = Union[EventCallbackWithEvent, EventCallbackWithoutEvent]
```

Comment:
If you have better ideas for this right now, I am all ears

Reply:
@mattpap unless you have any quick suggestions, this is ready to final review/merge

Reply:
Not really, this looks fine at least for now.